discord.sh

```diff
@@ -5,7 +5,15 @@ socat $SOCAT_ARGS \
     UNIX-CONNECT:$XDG_RUNTIME_DIR/discord-ipc-0 \
     &
 socat_pid=$!
+
+FLAGS='--enable-gpu-rasterization --enable-zero-copy --enable-gpu-compositing --enable-native-gpu-memory-buffers --enable-oop-rasterization --enable-features=UseSkiaRenderer,CanvasOopRasterization "$@"'
```

Comment:
I think it would be better not to mix flags that we set with user-provided args, i.e. move `"$@"` from here to the invocation line.